Homework/Frozenset.py

```diff
@@ -0,0 +1,22 @@
+steps_1 = ["open browser", "navigate to page", "click login"]
+steps_2 = ["open browser", "navigate to page", "fill form"]
+steps_3 = ["open browser", "navigate to page", "click login"]  # duplicate steps
+
+test_scenarios = {
+    "test_case_1": frozenset(steps_1),
+    "test_case_2": frozenset(steps_2),
+    "test_case_3": frozenset(steps_3)
+}
+
+
+def check_scenario_exists(test_case_name, test_scenarios):

```

Comment:
Shadows name 'test_scenarios' from outer scope. Better is using the other parameter name